packages/components/src/Filter/FilterButton/FilterButtonRemovable/FilterButtonRemovable.tsx

```diff
@@ -25,12 +26,20 @@ export const FilterButtonRemovable = forwardRef<
   FilterButtonRemovableRefs,
   FilterButtonRemovableProps
 >(({ triggerButtonProps, removeButtonProps, ...restProps }, ref) => {
+  const { formatMessage } = useIntl()
   const customRefObject = isRefObject(ref) ? ref.current : null
   const removeButtonRef = customRefObject?.removeButtonRef
 
-  const removeButtonLabel =
-    removeButtonProps?.tooltipText ??
-    `Remove filter - ${triggerButtonProps?.label}`
+  const removeButtonLabel = formatMessage(
+    {
+      id: "filterButtonRemovable.tooltipText",
+      defaultMessage: "Remove filter - {filter}",
+      description: "Label for the tooltip used in a filter remove button",
+    },
+    {
+      filter: removeButtonProps?.tooltipText || triggerButtonProps?.label,
+    }
+  )
```

Comment:
I can't say I'm too familiar with `formatMessage`, but is this the equivalent of writing this?

```
const removeButtonLabel =
    removeButtonProps?.tooltipText ??
    formatMessage({ id: "filterButtonRemovable.tooltipText" ... })
```

If so, what's the benefit of the format? Just wondering since it makes `formatMessage` always get called as opposed to only when `tooltipText` doesn't exist.

Also noticed that the optional chaining on `triggerButtonProps.label` isn't necessary since it's actually a required field.

Reply:
The way I made it is we would always have `Remove filter` by default in the message, even when passing a tooltip text, but I just realised it might be to strict.

Reply:
Actually, this is necessary for the change I intended. If we pass a `tooltipText`, which is the case of the `FilterBarMultiSelect` passing down the filter name instead of the custom label (explained [here](https://github.com/cultureamp/kaizen-design-system/pull/4604#discussion_r1552733743)).